features/step_definitions/issues_steps.rb

```diff
@@ -1,43 +1,21 @@
-And(/^I open New Issue creation form$/) do
-  issue_add_new_click
-end
-
 When(/^I submit New Issue form with valid data and issue type (.*)$/) do |issue_type|
-  issue_create_issue_fill_form(issue_type)
-  issue_create_issue_commit
-end
-
-Then(/^I see successfully created issue$/) do
-  @wait.until {@driver.find_element(:css => '.subject h3').displayed?}
-
-  expected_issue_subj = @issue_subj
-  actual_issue_subj = @driver.find_element(:css => '.subject h3').text
-  expect(actual_issue_subj).to eql(expected_issue_subj)
+  @issue_subject = on(ProjectSettingsPage).add_new_issue(issue_type)
 end
 
 And(/^I open Issues page$/) do
-  open_page_issues
+  on(ProjectSettingsPage).issues
 end
 
 Then(/^I see created issue in the list$/) do
-  @wait.until {@driver.find_element(:css => '.subject a').displayed?}
-  issue_added_resolve_name
-
-  expected_visible_issue = @issue_subj
-  actual_visible_issue = @driver.find_elements(:css => '.subject a')[@issue_index].text
-  expect(actual_visible_issue).to eql(expected_visible_issue)
+  result = on(ProjectSettingsPage).issue_subjects_elements.map(&:text).include? @issue_subject
+  expect(result).to be(true)
```

Comment:
should look like this:


```ruby
expect(issue_subjects).to include @issue_subject
```

this will allow you to find error causes in future

```ruby
expect(result).to be(true)
```
won't help you here